tests/integration.rs

```diff
@@ -262,6 +262,44 @@ fn looping() -> Result<()> {
     .run()
 }
 
+#[test]
+fn open() -> Result<()> {
+  env::set_var("DEGENERATE_OPEN_COMMAND", "echo");
```

Comment:
This will set the env var in the current process, which we should avoid, since it will conflict with other tests. Can we reuse the `Test` struct? We could add an `env_var` method:

```
fn env_var(mut self, key: &str, value: &str) -> Self {
  self.env_vars.push((key, value));
  self
}
```

And then set them when the command runs.